webpack.config.js

```diff
@@ -17,7 +18,7 @@ const staticPath = path.join(distPath, 'static');
 const webjackJsHack = ".v2.";
 const plugins = [
     new MonacoEditorWebpackPlugin({
-        languages: ['cpp', 'go', 'rust', 'swift', 'pascal'],
+        languages: [ 'cpp', 'go', 'pascal', 'python', 'rust', 'swift' ],
```

Comment:
Minor potential gotcha here. Maybe add a note to `AddingALanguage.md` to say "if you're adding a language using a built-in Monaco language mode, it needs to be bundled in the webpack config" if you think it fits.